rustls/src/tls13/key_schedule.rs

```diff
@@ -463,6 +464,7 @@ impl KeyScheduleTraffic {
 
     pub(crate) fn update_encrypter(&mut self, common: &mut CommonState) {
```

Comment:
Given this one now sends a message (which makes it uniquely specific to `KeyUpdate` usage) maybe we could vary its naming to break the implied symmetry with `update_decrypter`? Perhaps `update_encrypter_for_key_update`?

Reply:
Renamed it to `update_encrypter_and_notify()`, what do you think?

Reply:
Sounds good!